cf-log/cf-log-stack.ts

```diff
@@ -0,0 +1,274 @@
+                      {"name": "request_ip", "type": "varchar"},
+                      {"name": "method", "type": "varchar"},
+                      {"name": "host", "type": "varchar"},
+                      {"name": "uri", "type": "varchar"},
+                      {"name": "status", "type": "integer"},
+                      {"name": "referrer", "type": "varchar"},
+                      {"name": "user_agent", "type": "varchar"},
+                      {"name": "query_string", "type": "varchar"},
+                      {"name": "cookie", "type": "varchar"},
+                      {"name": "result_type", "type": "varchar"},
+                      {"name": "request_id", "type": "varchar"},
+                      {"name": "host_header", "type": "varchar"},
+                      {"name": "request_protocol", "type": "varchar"},
+                      {"name": "request_bytes", "type": "bigint"},
+                      {"name": "time_taken", "type": "real"},
+                      {"name": "xforwarded_for", "type": "varchar"},
+                      {"name": "ssl_protocol", "type": "varchar"},
+                      {"name": "ssl_cipher", "type": "varchar"},
+                      {"name": "response_result_type", "type": "varchar"},
+                      {"name": "http_version", "type": "varchar"},
+                      {"name": "fle_status", "type": "varchar"},
+                      {"name": "fle_encrypted_fields", "type": "integer"},
+                      {"name": "c_port", "type": "integer"},
+                      {"name": "time_to_first_byte", "type": "real"},
+                      {"name": "x_edge_detailed_result_type", "type": "varchar"},
+                      {"name": "sc_content_type", "type": "varchar"},
+                      {"name": "sc_content_len", "type": "bigint"},
+                      {"name": "sc_range_start", "type": "bigint"},
+                      {"name": "sc_range_end", "type": "bigint"},
+                      {"name": "year", "type": "varchar"},
+                      {"name": "month", "type": "varchar"},
+                      {"name": "day", "type": "varchar"},
+                      {"name": "hour", "type": "varchar"},
+                      {"name": "file", "type": "varchar"}
+                    ]
+                  }`), ' */')
+            }
+        })
+
+        const p0 = new iam.PolicyStatement({
+            effect: iam.Effect.ALLOW,
+            actions: ["athena:StartQueryExecution", "athena:GetQueryExecution"],
+            resources: ['*']
+        });
+        const p1 = new iam.PolicyStatement({
+            effect: iam.Effect.ALLOW,
+            actions: ["s3:ListBucket", "s3:GetBucketLocation"],
+            resources: [CloudFrontAccessLogsBucket.bucketArn]
+        });
+        const p2 = new iam.PolicyStatement({
+            effect: iam.Effect.ALLOW,
+            actions: ["s3:PutObject", "s3:GetObject"],
+            resources: [CloudFrontAccessLogsBucket.arnForObjects("/*")]
```

Comment:
For least privilege, the transformer just need read logs from `gz prefix` then write objects to `parque prefix`.